src/redux/actions/currentWeather.js

```diff
@@ -0,0 +1,29 @@
+import { areRequestsCanceled } from '../../common/axios-config';
+import { ERRORS } from '../../common/constants';
+
+const PREFIX = 'CURRENT_WEATHER/';
+
+export const SET_CURRENT_SUCCESS = `${PREFIX}SET_CURRENT_SUCCESS`;
+export const SET_CURRENT_FAIL = `${PREFIX}SET_CURRENT_FAIL`;
+export const SET_CURRENT_LOADING = `${PREFIX}SET_CURRENT_LOADING`;
+export const SET_CURRENT_LOADED = `${PREFIX}SET_CURRENT_LOADED`;
+
+const setCurrentFail = error => ({ type: SET_CURRENT_FAIL, payload: error });
```

Comment:
IMHO, I'd specify Current with sometinhg so that it is obvious that things are about CurrentWeather not something elxe current, like request, for instance.